Updates overview.md (#20329)
dotnet/AspNetCore.Docs: aspnetcore/security/data-protection/configuration/overview.md

## ProtectKeysWithAzureKeyVault

> Log in to Azure using the CLI, you can type:

Reviewer: ```suggestion
Log in to Azure using the CLI, for example:
```
